x/authz/simulation/operations_test.go

```diff
@@ -60,9 +60,9 @@ func (suite *SimTestSuite) TestWeightedOperations() {
 		// the following checks are very much dependent from the ordering of the output given
 		// by WeightedOperations. if the ordering in WeightedOperations changes some tests
 		// will fail
-		suite.Require().Equal(expected[i].weight, w.Weight(), "weight should be the same")
-		suite.Require().Equal(expected[i].opMsgRoute, operationMsg.Route, "route should be the same")
-		suite.Require().Equal(expected[i].opMsgName, operationMsg.Name, "operation Msg name should be the same")
+		suite.Require().Equal(expected[i].weight, w.Weight(), "test: %d, weight should be the same", i)
+		suite.Require().Equal(expected[i].opMsgRoute, operationMsg.Route, "test: %d, route should be the same", i)
+		suite.Require().Equal(expected[i].opMsgName, operationMsg.Name, "test: %d, operation Msg name should be the same", i)
```

Comment:
we were not checking the results, all tests were failing , and wrong route was used.... let's solve it in other issue

Reply:
https://github.com/cosmos/cosmos-sdk/issues/11050